Client/Frontend/Browser/Browser.swift

```diff
@@ -153,10 +159,36 @@ class Browser: NSObject, WKScriptMessageHandler {
             webView.alpha = 1.0
         }
     }
+
+    var bars = [SnackBar]()
+    func addSnackbar(bar: SnackBar) {
+        bars.append(bar)
+        delegate?.snackbarAdded(bar)
+    }
+
+    func removeSnackbar(bar: SnackBar) {
+        if let index = find(bars, bar) {
+            bars.removeAtIndex(index)
+            delegate?.snackbarRemoved(bar)
+        }
+    }
+
+    func removeAllSnackbars() {
+        for bar in bars {
```

Comment:
I don't know if this will work because you are modifying the array that you are looping over. This is a common error to do in Objective-C with `NSMutableArray` and since a Swift array is basically mapped on top of `NSMutableArray` I would not be surprised if it would raise similar errors.